public/pages/Monitors/containers/Monitors/Monitors.js

```diff
@@ -386,7 +384,7 @@ export default class Monitors extends Component {
       pageIndex: page,
       pageSize: size,
       pageSizeOptions: DEFAULT_PAGE_SIZE_OPTIONS,
-      totalItemCount: Math.min(MAX_MONITOR_COUNT, totalMonitors),
+      totalItemCount: totalMonitors,
```

Comment:
The table loads the `totalItemCount` items in memory right? Have we checked how this behaves now that this could potentially be unbounded. If we don't have a max for the `MAX_MONITOR` backend setting then a search for total Monitors could return up to `10,000` Monitors (I believe that's the natural limitation of the search we do).

Have we tried to populate a table this large and seen how it scales? It's possible this limitation was made at the time to safeguard the frontend.